tsconfig.json

```diff
@@ -8,6 +8,9 @@
         "moduleResolution": "node",
         "sourceMap": true,
         "outDir": "./dist",
+        "types": ["node", "express", "multer", "mocha", "jest", "express-session"],
```

Comment:
Are the config changes needed?

Reply:
Yes, this throws errors when npm test is ran without jest, mocha, and express-session being included

Reply:
for tom?